src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PremisesController.kt

```diff
@@ -121,6 +121,7 @@ class PremisesController(
 
         summaries.map(premisesSummaryTransformer::transformDomainToApi)
       }
+      else -> throw RuntimeException("Could not determine service")
```

Comment:
Would prefer just adding the missing enum value rather than an else so it's easy to see where changes potentially need to be made if another service is ever added (I know there's only CAS1/2/3 so this is unlikely but still), i.e.

```
ServiceName.cas2 -> throw RuntimeException("cas2 not supported")
```

Reply:
good shout on handling the missing enum value 👍 

I don't _think_ we'll have the concept of 'premises' in CAS-2, at least for foreseeable future - a third party called NACRO will be handling everything after the apply stage, we've got no plans to show premises at the moment.